modules/oadp-creating-restore-cr.adoc

```diff
@@ -55,3 +55,71 @@ $ oc get restore -n openshift-adp <restore> -o jsonpath='{.status.phase}'
 $ oc get all -n <namespace> <1>
 ----
 <1> Namespace that you backed up.
+
+. If you use Restic to restore `DeploymentConfig` objects or if you use post-restore hooks, run a cleanup script by entering the following command:
```

Comment:
I think that this could be presented more clearly, but at least tell the user that they need to save and run the script that you're providing. It's currently only implied that the script is _the_ cleanup script.

Reply:
@kalexand-rh 
The user doesn't need to save the script - it already exists in the system as `dc-restic-post-restore.sh`. The SME wanted to provide it here for reference purposes. I've changed the title of the collapsible block to make it clearer.

Reply:
@kalexand-rh If you approve, I'll squash my commits.

Reply:
@sbeskin-redhat I do not see @kalexand-rh approval. I would feel more comfortable with her OK on this PR. Can this wait until after Recharge week?

Reply:
@mburke5678 
I guess it can.  Is there any chance she will see this PR today?

Reply:
I still think it's not clear that the script _is already there_ and that no additional action is required from the user.

Reply:
@kalexand-rh  Please see it now. Is it clear enough?